src/sysext/sysext.c

```diff
@@ -508,6 +575,24 @@ static int mount_overlayfs(
         if (noexec >= 0)
                 SET_FLAG(flags, MS_NOEXEC, noexec);
 
+        if (upper_dir) {
+                r = append_overlayfs_path_option(&options, ",", "upperdir", upper_dir);
+                if (r < 0)
+                        return r;
+
+                SET_FLAG(flags, MS_RDONLY, false);
```

Comment:
SET_FLAG() is a bit weird to use if the last argument is a constant value. maybe just rewrite this as:

```c
flags &= ~MS_RDONLY;
```

that's shorter and i think most people will understand the pattern better

Reply:
Done.